### PR TITLE
fix: util.parseArgs() missing node:process import

### PR DESCRIPTION
**Pull request by javihernant**
fix parseArgs() not working due to missing import of node:process

this commit fixes issue #22363

### CODE REVIEW

**Comment by mmastrac** [comment]
This looks good, but can you try enabling the `test-parse-args.mjs` test by removing the line in `tests/node_compat/config.jsonc`?

See the notes in `tools/node_compat/TODO.md` for more info on what to do to update the test config files.

**Comment by javihernant** [comment]
Removing the line? You mean adding it? It is already there. I wonder, if the test was already there, how come it has never failed?

**Comment by mmastrac** [comment]
That file is a list of ignored tests, so removing it will run it

**Comment by kt3k** [comment]
@mmastrac @javihernant `tests/node_compat/config.jsonc` includes both ignore-list and include-list (`root.ignore` is for ignoring and `root.test` is for including), and `test-parse-args.mjs` is included and the test cases in it actually are passing.

You can check it with the command:
```
./target/debug/deno test --config tests/node_compat/deno.json -A tests/node_compat/test.ts -- test-parse-args.mjs
```

The fixed code path (the case when `args` option is omitted) doesn't seem tested in the compat test case.

I think we need to add our own test case in `tests/unit_node/util_test.ts` to exercise this fix.

**Comment by javihernant** [comment]
okay! Im going to try creating a custom test

File: tests/unit_node/util_test.ts [hunk 1]
```diff
@@ -315,3 +315,10 @@ Deno.test({
     );
   },
 });
+
+Deno.test({
+  name: "[util] parseArgs() with no args works",
+  fn() {
+    util.parseArgs();
```
Comment (kt3k) [inline tests/unit_node/util_test.ts:322]: This looks causing a type error. I think we need to pass an empty object
```suggestion
    util.parseArgs({});
```

Comment (javihernant) [reply tests/unit_node/util_test.ts]: yep, that was the problem. I've fixed it in my fork. I've seen the CI test that fails is run with `cargo test`. However, that takes a lot of time to run in my machine. What's the most appropriate way I should be running tests? I've used previously the `./target/debug/deno test --config tests/config/deno.json -A tests/unit_node/util_test.ts` command, which run only the tests of my PR. However, that didn't check for TS syntax error it seems.

**Review by kt3k — APPROVED** [review]
LGTM. Nice work!

**Pull request merged**